### ISSUE TITLE
`Programming exercises`: score in course exercise overview is not correct

### ISSUE THREAD

**Issue by HotamjonAminov** [ls1intum/Artemis#6589]
### Describe the bug

It shows 0% on task B Blatt 1, even though everything was fine when it was delivered. In addition, if you refresh the web page, you get 100% again. After going into the task to look at the overview of levies, there is no case with 0%, the last submission is 100%.

### To Reproduce

1. On the course exercise overview page the exercise shows 0%
1. After clicking on the exercise detail page and the feedback detail on that page the score is shown as 100%
1. Navigating back to the course overview page the score is visible as 100% (as it should be)

### Expected behavior

The score is not shown correctly in the course exercise overview page, where it is visible as 0% (possibly last result) instead of 100%

### Which version of Artemis are you seeing the problem on?

6.1.7

### What browsers are you seeing the problem on?

Chrome

**Comment by krusche** [comment]
Could you please post a screenshot or screencast of your issue. This would make it easier for us to understand and reproduce the issue. Thanks

**Comment by krusche** [comment]
@simon-hng could you please take a look at this issue?

**Comment by simon-hng** [comment]
Yes but I need additional information. @HotamjonAminov add screenshots and provide us with the following information so that we can reproduce it:

- What page/view does the bug appear on?
- In which course does this bug appear?

**Comment by HotamjonAminov** [comment]
I made little video from the issue, which shows it better. I hope it will help u )
[Screen Recording - Made with FlexClip.webm](https://github.com/ls1intum/Artemis/assets/69870480/15889096-3cd3-4599-a053-eb1a22618702)

**Comment by simon-hng** [comment]
Hey @HotamjonAminov thanks a lot for the screenshare!

I updated the issue name as well as the steps to reproduce so that other developers can understand it easier.

This issue was not reproducible and seems to stem from a caching issue. If you still want me to look further into it I recommend we set up a meeting and so we can check some things in the dev tools and see if we find a hint regarding what went wrong

**Comment by HotamjonAminov** [comment]
Hi, yes of course. I can on Friday, Monday (at 18 o'clock)

**Comment by simon-hng** [comment]
> Hi, yes of course. I can on Friday, Monday (at 18 o'clock)

Great send me an email to [simon.huang@outlook.de](mailto:simon.huang@outlook.de) so I can reply with a meeting link
